src/Castle.Facilities.AspNetCore.Tests/WindsorRegistrationExtensionsAcceptanceTestCase.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2004-2018 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Facilities.AspNetCore.Tests
+{
+	using System;
+
+	using Castle.Facilities.AspNetCore.Tests.Fakes;
+	using Castle.MicroKernel.Registration;
+	using Castle.Windsor;
+
+	using NUnit.Framework;
+
+	[TestFixture, Order(0)]
+	public class WindsorRegistrationExtensionsAcceptanceTestCase
+	{
+		[Test, Order(0)]
+		public void Should_throw_for_CrossWire_Registration_without_CrossWiresInto_being_called_first()
+		{
+			using (var container = new WindsorContainer())
+			{
+				Assert.Throws<InvalidOperationException>(() => { container.Register(Component.For<AnyComponent>().CrossWired()); });
+			}
```

Comment:
@generik0 

Your suggestion of a helpful exception was implemented here. Thanks!